modules/resources/deployment-scripts-import-acr/main.json

```diff
@@ -0,0 +1,164 @@
+        "OnSuccess",
+        "OnExpiration",
+        "Always"
+      ]
+    }
+  },
+  "resources": [
+    {
+      "type": "Microsoft.ManagedIdentity/userAssignedIdentities",
+      "apiVersion": "2018-11-30",
+      "name": "[parameters('managedIdName')]",
+      "location": "[parameters('location')]"
+    },
+    {
+      "copy": {
+        "name": "rbac",
+        "count": "[length(parameters('rbacRolesNeeded'))]"
+      },
+      "type": "Microsoft.Authorization/roleAssignments",
+      "apiVersion": "2020-08-01-preview",
+      "scope": "[format('Microsoft.ContainerRegistry/registries/{0}', parameters('acrName'))]",
+      "name": "[guid(resourceId('Microsoft.ContainerRegistry/registries', parameters('acrName')), parameters('rbacRolesNeeded')[copyIndex()], resourceId('Microsoft.ManagedIdentity/userAssignedIdentities', parameters('managedIdName')))]",
+      "properties": {
+        "roleDefinitionId": "[resourceId('Microsoft.Authorization/roleDefinitions', parameters('rbacRolesNeeded')[copyIndex()])]",
+        "principalId": "[reference(resourceId('Microsoft.ManagedIdentity/userAssignedIdentities', parameters('managedIdName'))).principalId]",
+        "principalType": "ServicePrincipal"
+      },
+      "dependsOn": [
+        "[resourceId('Microsoft.ManagedIdentity/userAssignedIdentities', parameters('managedIdName'))]"
+      ]
+    },
+    {
+      "copy": {
+        "name": "createImportImage",
+        "count": "[length(parameters('images'))]"
+      },
+      "type": "Microsoft.Resources/deploymentScripts",
+      "apiVersion": "2020-10-01",
+      "name": "[format('ACR-Import-{0}-{1}', parameters('acrName'), replace(replace(parameters('images')[copyIndex()], ':', ''), '/', '-'))]",
+      "location": "[parameters('location')]",
+      "identity": {
+        "type": "UserAssigned",
+        "userAssignedIdentities": {
+          "[format('{0}', resourceId('Microsoft.ManagedIdentity/userAssignedIdentities', parameters('managedIdName')))]": {}
+        }
+      },
+      "kind": "AzureCLI",
+      "properties": {
+        "forceUpdateTag": "[parameters('forceUpdateTag')]",
+        "azCliVersion": "[parameters('azCliVersion')]",
+        "timeout": "[parameters('timeout')]",
+        "retentionInterval": "[parameters('retention')]",
+        "environmentVariables": [
+          {
+            "name": "acrName",
+            "value": "[parameters('acrName')]"
+          },
+          {
+            "name": "imageName",
+            "value": "[parameters('images')[copyIndex()]]"
+          },
+          {
+            "name": "initialDelay",
+            "value": "[parameters('initialScriptDelay')]"
+          }
+        ],
+        "scriptContent": "      #!/bin/bash\r\n      set -e\r\n\r\n      echo \"Waiting on RBAC replication ($initialDelay)\"\r\n      sleep $initialDelay\r\n\r\n      echo \"Importing Image: $imageName into ACR: $acrName\"\r\n      az acr import -n $acrName --source $imageName --force\r\n    ",
```

Comment:
The `\r\n` must be causing the outdated error in the `brm validate` step in the CI. The line endings of the `main.bicep` file   are normalized to `\n` once it's checked in, so the temp `main.json` file that `brm validate` generates has `\n` instead of `\r\n` in the `scriptContent` string. The workaround is to change the line endings of `main.bicep` to LF locally and regenerate `main.json` to fix `scriptContent` and `templateHash`.

Reply:
I'll add a `.gitattributes ` file to normalize the line endings of bicep files.